common/ast/src/parser/expr.rs

```diff
@@ -291,6 +294,7 @@ impl<'a, I: Iterator<Item = WithSpan<'a, ExprElement<'a>>>> PrattParser<I> for E
             ExprElement::MapAccess { .. } => Affix::Postfix(Precedence(10)),
             ExprElement::IsNull { .. } => Affix::Postfix(Precedence(17)),
             ExprElement::Between { .. } => Affix::Postfix(Precedence(BETWEEN_PREC)),
+            ExprElement::IsDistinctFrom { .. } => Affix::Infix(Precedence(20), Associativity::Left),
```

Comment:
Is 20 BETWEEN_PREC? (IIRC)

Reply:
Yes
https://github.com/datafuselabs/databend/blob/c0d05f18ee4dea6c662539d8e0414ad87e938a6f/common/ast/src/parser/expr.rs#L36
 I will fix it ~